Tasks/UpdateConfigurationTask.php

```diff
@@ -21,24 +21,28 @@ public function __construct(ConfigurationRepository $repository)
 
     public function run($type, array $data)
     {
-        $configurable_id=null;
+        $configurable_id = null;
         try {
             if ($type == "user") {
-                $configurable_id=Auth::user()->id;
+                $configurable_id = Auth::user()->id;
 
-            } else if($type == "tenant") {
-                $configurable_id=Auth::user()->tenant_id;
+            } else if ($type == "tenant") {
+                $configurable_id = Auth::user()->tenant_id;
             }
-            $configuration = $this->repository->where('configurable_id',$configurable_id)->first();
-            if(!$configuration){
+            $configuration = $this->repository->where('configurable_id', $configurable_id)->first();
+
+            if (!$configuration) {
 
                 throw new NotFoundException();
             }
 
-            $history = app(CreateConfigurationHistoryTask::class)->run([$configuration->id,$configuration->configuration]);
```

Comment:
Calling task within other task, is not the correct way. Call create tenant history in action or controller (controller is more better)